README.md

```diff
@@ -44,7 +57,7 @@ Linux For Everyone has a good demo video [here](https://www.youtube.com/watch?v=
 
 Unpack the `tgz` file, into your home directory.
 
-    tar -C $HOME -xzf NoiseTorch_x64.tgz
+    tar -C $HOME -xzf NoiseTorch_x64_v0.12.0.tgz
```

Comment:
I've seen this - the moment after I pushed the tag :face_with_spiral_eyes: 
Thanks for adding it.